indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -0,0 +1,322 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import timeTableURL from 'indico-url:event_registration.api_event_timetable';
+
+import _ from 'lodash';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+import {useSelector} from 'react-redux';
+import {Checkbox, Form, Accordion, Icon} from 'semantic-ui-react';
+
+import {
+  FinalCheckbox,
+  FinalRadio,
+  FinalField,
+  FinalInput,
+  validators as v,
+} from 'indico/react/forms';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {Translate, Param, Plural, PluralTranslate, Singular} from 'indico/react/i18n';
+
+import '../../../styles/regform.module.scss';
+import {getStaticData} from '../selectors';
```

Comment:
move this import up a line: the css import above is considered "with side effects", so it's a boundary for automatic import ordering. But once you move this up the eslint rule for import sorting will automatically sort it in the right position

Reply:
Sorry Adrian, but after changing the import one line up, the linter fails after committing and the command `isort . --interactive` is not working to fix the import order, would you mind to tell me the command to run the indico eslinter? `which eslinter` neither works

Reply:
isort is for python, any formatting etc in javascript is handled by eslint. If you don't have eslint integration in your editor (in vscode that works almost out of the box), simply run `npx eslint --fix indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx`

Reply:
I have eslit installed in my vscode, but it is not syncing with the repository config I think, since, for instance, it does not gives suggestions in this case, but it gives others that don't apply for this project.